models/jules/R/model2netcdf.JULES.R

```diff
@@ -32,19 +32,19 @@ model2netcdf.JULES <- function(outdir) {
                 row.names = TRUE,
                 quote = FALSE)
     
-    vars <- ncdf4.helpers::nc.get.variable.list(nc)
+    vars <- names(nc[['var']])
     # Check that frac is reported
     if("frac_grid" %in% vars){
       frac <- ncdf4::ncvar_get(nc, "frac_grid")
     } else {
       PEcAn.logger::logger.warn("Surface type fraction is not an output and thus other outputs may not be parseable")
     }
     
-    base_dims <- ncdf4.helpers::nc.get.dim.names(nc, "GPP") # This isn't the best example, except that GPP is the default with read.output (and it is not reported by surface type)
+    base_dims <- nc$var[["GPP"]]$dim[[1]]$name # This isn't the best example, except that GPP is the default with read.output (and it is not reported by surface type)
```

Comment:
I don't think this will work. Based on the `ncdf4.helpers` source, you probably need something like:

```r
vapply(f$var[["GPP"]]$dim, `[[`, character(1), "name")
# ...or, perhaps more clearly...
unlist(lapply(f$var[["GPP"]]$dim, function(x) x$name))
```